src/games/controller.ts

```diff
@@ -60,22 +59,18 @@ export default class GameController {
     game.status = 'started'
     await game.save()
 
-    const player = await Player.create({
+    await Player.create({
       game,
       user,
       symbol: '2'
     }).save()
 
-    const gameToSend = await Game.findOneById(game.id)
-    if (!gameToSend) throw new BadRequestError(`Game does not exist`)
-    //console.log(getGuessBoard(gameToSend.board1))
-
     io.emit('action', {
       type: 'UPDATE_GAME',
-      payload: {...gameToSend, board1: getGuessBoard(gameToSend.board1)}
+      payload: gameToSend(game)
     })
 
-    return player
+    return {board:game.board2, guessBoard:game.board1}
```

Comment:
haha! yes board1 contains the ships of player1, and since this request will be made by player2, we put it in guessboard. As before, we can also just send either board1 or board2 as they are still empty but like this, it's more clearer :smile:

Reply:
yeah makes sense. also then it's always specified who the player is